response_operations_ui/cloud/cloudfoundry.py

```diff
@@ -0,0 +1,16 @@
+import cfenv
+from flask import current_app
+
+
+class ONSCloudFoundry(object):
+
+    def __init__(self):
+        self._cf_env = cfenv.AppEnv()
+
+    @property
+    def detected(self):
```

Comment:
An ONSCloudFoundry object could probably just evaluate to True if the environment is detected, rather than passing back the app attribute:

```Python
def __bool__(self):
    return True if self._cf_env.app
```

Reply:
Then do

```Python
cf = ONSCloudfoundry()
if cf:
```